### ISSUE TITLE
Disable some extensions of Nintendo Wii remote

### ISSUE THREAD

**Issue by sltvtr** [dkosmari/hid-wiimote-plus#1]
Hello,
I don't know if this is the right place to ask, but I'll try :)
The original kernel module, and the one in this repo, both work (from my perspective), but also have the same "issue".

The controller attached to guitar presents itself in linux like one bluetooth device, but with 5 different inputs:

```
/dev/input/event11:     Nintendo Wii Remote Accelerometer
/dev/input/event12:     Nintendo Wii Remote IR
/dev/input/event13:     Nintendo Wii Remote
/dev/input/event14:     Nintendo Wii Remote Guitar
/dev/input/event15:     Nintendo Wii Remote Motion Plus
```

The "Motion Plus" extension keeps constantly sending signals which makes guirat almost unusable - can't map keys in the game (clone hero), even if somehow I did map those keys it interfers with eg. button long-press.
Long story short  - is there some "easy" way of disabling that extension modifying the code? 
For now my only solution/workaround is manualy deleting file /dev/input/event15 after connecting the controller.

**Comment by dkosmari** [comment]
Check out the udev rule file, `99-wiimote.rules`, that was installed into `/etc/udev/rules.d/`. Replace the "Nintendo Wii Remote Motion Plus" line with this:

```udev
SUBSYSTEM=="input", ATTR{name}=="Nintendo Wii Remote Motion Plus", ATTR{inhibited}="1"
```

(Note the use of `ATTR`, and not `ATTRS`, because not all devices in the chain have the "inhibited" attribute. If you use `ATTRS` here you'll get some harmless errors reported on the journal, but it should still work.)
The updated rule should take effect after you disconnect and reconnect the controller.

That said, games should not be reading inputs tagged with `INPUT_PROP_ACCELEROMETER` unless they're prepared to handle accelerometer and gyro data. Maybe you can report it to them as a bug.

Some games also like to read all available input devices, even the ones not being used. But this increases power consumption. So if you don't plan on using accelerometers, gyro, IR camera, it might be better to inhibit them in the udev rule.

**Comment by sltvtr** [comment]
Thank You!
That was exactly was I was looking for, this solution works even on native linux kernel module.
Tinkering with udev was my first idea, but I couldn't find anything reliable/working up to this point.
Your rule works perfectly, the device is registered as before (all 5 inputs/events) but all input from Motion Plus is supressed, evtest is silent, and everything works flawlessly :)
And as You suggested I disabled all not-needed inputs using this method.
Best Regards!

**Comment by dkosmari** [comment]
Commit 966a3ed now has this into the documentation.
